ert3/workspace/__init__.py

```diff
@@ -1,5 +1,6 @@
 from ert3.workspace._workspace import initialize
 from ert3.workspace._workspace import load
+from ert3.workspace._workspace import get_experiments
```

Comment:
I suggest that we rename the function to `get_experiment_names`, to make it clear that it is the names and not the entire experiments it is returning...

Reply:
Also, this function needs tests